internal/driver/glfw/loop.go

```diff
@@ -38,7 +38,7 @@ func runOnMain(f func()) {
 func runOnMainWithWait(f func(), wait bool) {
 	// If we are on main just execute - otherwise add it to the main queue and wait.
```

Comment:
I think this comment should be updated, something like

// If the app main loop is not yet running, run f directly only if we are on the main goroutine.